bloptools/tests/test_bayesian_shadow.py

```diff
@@ -33,9 +33,9 @@ def test_bayesian_agent_tes_shadow(RE, db, shadow_tes_simulation):
         db=db,
     )
 
-    RE(agent.initialize(acqf="qr", n_init=4))
+    RE(agent.initialize("qr", n_init=4))
 
-    RE(agent.learn(acqf="ei", n_iter=2))
-    RE(agent.learn(acqf="pi", n_iter=2))
+    RE(agent.learn("ei", n_iter=2))
+    RE(agent.learn("pi", n_iter=2))
```

Comment:
All these options need to be listed in the general documentation (in Intro section, not in tutorials (=notebooks)).

Reply:
I'll make a PR with improved documentation outside the tutorials, for this and the above